docs/src/main/paradox/recordio.md

```diff
@@ -0,0 +1,61 @@
+# RecordIO Framing
+
+The codec parses an incoming RecordIO-formatted stream of ByteStrings into distinct frames.
```

Comment:
is `RecordIO` a well know term? Perhaps clarify what it is? Perhaps give an example of what it is, such as `JSON`?